### PR TITLE
Dynamic post

### PR DESCRIPTION
**Pull request by Chriscastanuela**
### Description
* Makes the post component dynamic so that it is different on the `Your Posts` page
### How Has This Been Tested?
* Ran in DOM
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by GreyMatteOr — APPROVED** [review]
Looks like the next step is just actually pulling down Posts from the DB

File: src/App.js [hunk 1]
```diff
@@ -9,10 +9,16 @@ import './Scss/base.scss';
...
+// Dummy Info
+import ringIcon from './images/ring-icon.png';
+import dummyIcon from './images/dummyIcon.png';
 const loremIpsum = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.';
 const lat = -888;
 const lon = -999;
 const name = 'John Doe';
+const dummyRing = 0
+const dummyDate = '07/04/1776'
+const dummyLastLike = 'Boston, MA'
```
Comment (MichaelEWalker87) [inline src/App.js:21]: Looks good

File: src/Post/Post.js [hunk 2]
```diff
@@ -63,26 +62,55 @@ export function Post(props) {
...
+                <section className='post-left'>
+                    <div className='post-left-top'>
+                        <img src={props.icon} alt='User Icon' id='user-icon'/>
+                    </div>
+                    <div className='post-left-bottom' style={{display: 'grid',
+                    gridTemplateRows: '1em 1em', paddingTop: '.5em'}}>
+                        <em style={{margin: 0, gridRowStart: 1, gridRowEnd: 1}}><h6  style={{margin: 0, gridRowStart: 1, gridRowEnd: 1}}>Lat: </h6></em>
+                        <em style={{margin: 0, gridRowStart: 1, gridRowEnd: 1}}><h6  style={{margin: 0, gridRowStart: 1, gridRowEnd: 1}}>{props.lat}</h6></em>
+                        <em style={{margin: 0, gridRowStart: 2, gridRowEnd: 2}}><h6  style={{margin: 0, gridRowStart: 2, gridRowEnd: 2}}>Lon: </h6></em>
+                        <em style={{margin: 0, gridRowStart: 2, gridRowEnd: 2}}><h6  style={{margin: 0, gridRowStart: 2, gridRowEnd: 2}}>{props.lon}</h6></em>
```
Comment (MichaelEWalker87) [inline src/Post/Post.js:77]: Make sure we can cut this down line size wise

File: src/Post/Post.js [hunk 3]
```diff
@@ -63,26 +62,55 @@ export function Post(props) {
...
-                </div>
+    if (props.myPostsPage) {
+        return (
+            <section className='post'>
+                <section className='post-left'>
+                    <div className='post-left-top'>
+                        <img src={props.icon} alt='User Icon' id='user-icon'/>
+                    </div>
+                    <div className='post-left-bottom' style={{display: 'grid',
+                    gridTemplateRows: '1em 1em', paddingTop: '.5em'}}>
```
Comment (MichaelEWalker87) [inline src/Post/Post.js:73]: Can we remove this in line style and place in an SCSS file

**Review by MichaelEWalker87 — APPROVED** [review]

**Pull request merged**